uefi_test.go

```diff
@@ -43,11 +44,34 @@ func TestAppendUEFIFirmwareDeviceConfig(t *testing.T) {
 		t.Errorf("Expected non-empty qemuParams, found %s", c.qemuParams)
 	}
 
-	expected := "-drive if=pflash,format=raw,readonly,file=OVMF_CODE.fd -drive if=pflash,format=raw,file=OVMF_VARS.fd"
+	expected := "-drive if=pflash,format=raw,readonly=on,file=OVMF_CODE.fd -drive if=pflash,format=raw,file=OVMF_VARS.fd"
 	result := strings.Join(c.qemuParams, " ")
 	if expected != result {
 		t.Fatalf("Failed to append parameters\nexpected[%s]\n!=\nfound   [%s]", expected, result)
 	}
 }
 
+func TestNewFirmwareDev(t *testing.T) {
```

Comment:
Note, while this will pass on a local system, it'll need to be commented out in the PR since CI won't have these files (at least until I either update the CI image to include OVMF, or sort out how to mock out local filesystem checks).